### PR TITLE
Minutes for the 2021-12-09

### PR DESCRIPTION
**Pull request by SimonBiggs**

### CODE REVIEW

File: minutes/2021-12-09.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# 2021-12-09
+
+- SB reached out to from Innolitics about RDM example repo
+- Interested in joining efforts but wait until some progress has been made
+- IEC-62304 requires use of a QMS, TGA doesn't require
```
Comment (SimonBiggs) [inline minutes/2021-12-09.md:5]: @pchlap, a change in wording 

```suggestion
- IEC-62304 requires use of a QMS. For exempt clinical decision support software a QMS is still needed, but the TGA does not require this QMS to be certified.
```

File: minutes/2021-12-09.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- ISO 13485 for QMS, GitHub can be used to fulfil
+- Linking between requirements and testing and code providing functionality
+- RDM can use Git hooks to achieve this (commits link to issues)
+- Is example repo still needed given overall groups shift?
```
Comment (SimonBiggs) [inline minutes/2021-12-09.md:9]:
```suggestion
- Is example repo still needed given overall group's shift?
```

**Review by SimonBiggs — COMMENTED** [review]
Hi @pchlap,

I just had a couple of comments on the minutes. Feel free to commit these suggestions if you're happy with my changes.

**Comment by pchlap** [comment]
Thanks @SimonBiggs

**Pull request merged**